src/tests/Microsoft.PowerFx.Interpreter.Tests/PADIntegrationTests.cs

```diff
@@ -121,6 +121,14 @@ public void DataTableEvalTest2()
         { Other : 5}
      )).Other");
             Assert.IsType<BlankValue>(result4);
+
+            var symbol = new SymbolTable();
+            symbol.EnableMutationFunctions();
+
+            engine.Config.SymbolTable = symbol;
+
+            var result5 = engine.Eval("Remove(robintable, {Names:\"name2\"});CountRows(robintable)", options: new ParserOptions() { AllowsSideEffects = true });
+            Assert.Equal(2, ((NumberValue)result5).Value);
         }
```

Comment:
>} [](http://example.com/codeflow?start=8&length=1)

Can you expand this to include calls to Patch() and Collect()? With similar asserts.